src/endian.cc

```diff
@@ -0,0 +1,7 @@
+#include <endian.h>
+
+#include <boost/endian/conversion.hpp>
+
+uint64_t htobe64(uint64_t host_64bits) {
+    return boost::endian::native_to_big(host_64bits);
```

Comment:
Actually I'd prefer if you put this directly in agent code (instead of htobe64).